doc/nrf/includes/sample_board_rows.txt

```diff
@@ -248,6 +248,10 @@
 
 | nRF54L20 PDK |   | :ref:`nrf54l20pdk <zephyr:nrf54l20pdk_nrf54l20>` | ``nrf54l20pdk/nrf54l20/cpuapp`` |
 
+.. nrf54l09pdk_nrf54l09_cpuapp
+
+| nRF54L09 PDK |   | :ref:`nrf54l09pdk <zephyr:nrf54l09pdk_nrf54l09>` | ``nrf54l09pdk/nrf54l09/cpuapp`` |
```

Comment:
This is an obsolete board target name. Also, I'm not sure if the Zephyr article for this should be available at all at this stage